plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -470,6 +470,14 @@ void QtOSGViewer::_UpdateViewerCallback()
                 }
             }
         }
+        if (!_excessGUIFunctions.empty()) {
```

Comment:
I would make here the only place to modify `listGUIFunctions` use [splice](https://en.cppreference.com/w/cpp/container/list/splice) instead of pushing to 2 different lists judging by size.